### ISSUE TITLE
Wildcard with alias has problem

### ISSUE THREAD

**Issue by cesaryuan** [codeque-co/codeque#35]
```js
(() => {
    function $$_a() {
        2
    }
})();
```
cannot match
```js
(() => {
    function func1() {
      1
    }
    function func2() {
      2
    }
})()
```
but if we replace `$$_a` in query to `$$`, it can match.

**Comment by jayu** [comment]
Thank you! This is another one I've spot myself recently as well.

I will fix this. 

The reason for the bug is that `$$_a` binds to `func1`, because it is earlier in the code than `func2`. This is incorrect binding, because the function body does not match. So binding is too optimistic :D

**Comment by jayu** [comment]
It has been fixed in extension version `0.29.0` and core version `0.6.1` 🥳
